lib/index.js

```diff
@@ -86,14 +86,19 @@ Optimizely.prototype.initialize = function() {
  */
 
 Optimizely.prototype.track = function(track) {
+  var opts = this.options;
   // Use the new-style API (which is compatible with Classic and X)
   var payload = {
     type: 'event',
     eventName: track.event(),
     tags: track.properties()
   };
 
-  if (payload.tags.revenue) {
+  if (payload.tags.revenue && opts.sendRevenueOnlyForOrderCompleted) {
```

Comment:
Revenue was previously sent alone as the 4th attribute but now is masked in eventProperties. 
https://github.com/segment-integrations/analytics.js-integration-optimizely/pull/37/files#diff-6d186b954a58d5bb740f73d84fe39073R115